lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1434,8 +1434,8 @@ Attribute MfmaEncodingAttr::parse(AsmParser &parser, Type type) {
 
 void MfmaEncodingAttr::print(AsmPrinter &printer) const {
   printer << "<{"
-          << "version = " << getVersionMajor() << "." << getVersionMinor()
-          << ", "
+          << "versionMajor = " << getVersionMajor() << ", "
+          << "versionMinor = " << getVersionMinor() << ", "
```

Comment:
Without this change, printing and parsing formats are different.
This prevent correct parsing of dumped IR, i.e. if CDNA 2 layout is dumped it looks like this:
`#triton_gpu.mfma<{version = 2.0, warpsPerCTA = [1, 1], instrShape = [32, 32], isTransposed = false}>`

parser reads this description, but fails to parse version field, so it generates layout that looks like this: `#triton_gpu.mfma<{versionMajor = 0, versionMinor = 0, warpsPerCTA = [1, 1], instrShape = [32, 32], isTransposed = false}>`

`versionMajor = 0` is invalid